Tests/UnitTests/Fatras/Digitization/BoundRandomValues.hpp

```diff
@@ -0,0 +1,125 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <array>
+
+namespace ActsFatras {
+
+/// Generate ranom numbers within the regular, centric Rectangle.
+struct RectangleRandom {
+  double xmax;
+  double ymax;
+
+  /// Constructor from half lengths.
+  ///
+  /// @param xmax_ the half length in x
+  /// @param ymax_ the half length in y
+  RectangleRandom(double xmax_, double ymax_) : xmax(xmax_), ymax(ymax_) {}
+
+  /// Given two random numbers @param r0 and @param r1
+  /// generate a x/y position inside the Rectangle @return
+  std::array<double, 2> operator()(double r0, double r1) const {
```

Comment:
I guess the `r0` and `r1` are assumed to be between [0, 1]? Will it be a problem if not?

Reply:
I will clarify, I do not want to overcheck in this kind of code.

Reply:
Yes, indeed.